lib/Platform/Intl/PlatformIntlApple.mm

```diff
@@ -81,6 +214,121 @@
   return std::u16string(u"uppered");
 }
 
+// Implementation of
+// https://tc39.es/ecma402/#sec-getoption
+// Funcion split into two depending on which data type is needed
+vm::CallResult<Option> getOptionBool(
```

Comment:
Since these are split, we can simplify this function:
1. It can return a `CallResult<bool>`.
2. The `type` parameter can be omitted, and we don't need to check it below.
3. The `fallback` parameter should just be a `bool`.